src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -223,7 +223,7 @@ tests:
       arguments:
           values:
               - name: "ms"
-                value: 3000
+                value: 1000
```

Comment:
Please make sure the YAML changes get reviewed by the cert folks.

@cjandhyala @raju-apple @manjunath-grl